alfalfa_web/components/Sites/Sites.js

```diff
@@ -9,72 +9,72 @@ import { StartDialog } from "./StartDialog";
 export const Sites = () => {
   const [loading, setLoading] = useState(true);
   const [selected, setSelected] = useState([]);
-  const [sites, setSites] = useState([]);
+  const [runs, setRuns] = useState([]);
   const [showErrorDialog, setShowErrorDialog] = useState(null);
   const [showPointDialog, setShowPointDialog] = useState(null);
   const [showStartDialog, setShowStartDialog] = useState(null);
 
   const validStates = {
-    start: ["ready"],
-    stop: ["preprocessing", "starting", "started", "running", "stopping"],
-    remove: ["ready", "complete", "error"],
-    download: ["ready", "complete", "error"]
+    start: ["READY"],
+    stop: ["PREPROCESSING", "STARTING", "STARTED", "RUNNING", "STOPPING"],
+    remove: ["READY", "COMPLETE", "ERROR"],
+    download: ["READY", "COMPLETE", "ERROR"]
```

Comment:
I got rid of the code which was lowercasing all of the statuses when they went through the api